### ISSUE TITLE
jsx-tag-spacing doesn't allow for forbid a space before ">" in a start tag

### ISSUE THREAD

**Issue by mmsbrggr** [yannickcr/eslint-plugin-react#1396]
In our project we have the need for forbidding spaces before the ">" symbol in a start tag. For example the following should not be allowed:
```javascript
<RectangleSelection
    minWidth={150}
    minHeight={50} >
    {children}
</RectangleSelection>
```

In comparison the following should not result in an error:
```javascript
<RectangleSelection
    minWidth={150}
    minHeight={50}>
    {children}
</RectangleSelection>
```

**Comment by jseminck** [comment]
Would you be up for making a PR for this feature? I had a quick look today, and it shouldn't be that complicated as the code will be quite similar to how the current `beforeSelfClosing` option works.

I'm ready to assist if needed.

We'll need to add a new option to support this though, I don't know what the name should be. Perhaps simply `beforeClosing` ?

**Comment by ljharb** [comment]
`beforeClosing` might be `</` - does the rule already handle `< `?

**Comment by cjskillingstad** [comment]
PR opened here: #1641

**Comment by ljharb** [comment]
Closed by #1641.

**Comment by danrot** [comment]
Any idea when this will be released?

**Comment by caub** [comment]
Yea, I still have this issue, it's not released?

```json
"react/jsx-tag-spacing": ["error", {
			"closingSlash": "never",
			"beforeSelfClosing": "always",
			"afterOpening": "never",
			"beforeClosing": "never"
		}],
```

```jsx
<Foo >foo</Foo >
```
is allowed

**Comment by ljharb** [comment]
It’s published in v7.6; are you sure you’re running a local eslint?

**Comment by caub** [comment]
@ljharb yes ` npm list -g --depth=0` doesn't show any `eslint` modules

if I write `<Searchbar  >` it propose me to fix to `<Searchbar >`, not `<Searchbar>` (which is also valid, but I'd like to enforce this convention)
